src/controller/index.ts

```diff
@@ -250,13 +250,11 @@ export default class SimulariumController {
             return Promise.reject(e);
         }
 
-        return this.simulator
-            .convertTrajectory(dataToConvert, fileType)
-            .then(() => {
-                if (this.simulator) {
-                    this.simulator.requestSingleFrame(0);
-                }
-            });
```

Comment:
Is this no longer needed because of the `simulariumController.changeFile` call in `Viewer.tsx`?

Reply:
Correct!